tests/acceptance/tests/Settings/BasicConsentBannerDeclineCookies.spec.ts

```diff
@@ -4,10 +4,9 @@ test('As a shop customer, I want to continue shopping without accepting the cook
     ShopCustomer,
     StorefrontHome,
     TestDataService,
-    DefaultSalesChannel,
 }) => {
 
```

Comment:
Do we need to skip the test in SaaS as well because of the system config value?

Reply:
yes

Reply:
we can test them again, after https://github.com/shopware/shopware/pull/5925 is merged and deployed to saas

Reply:
I will create a ticket that we enable as soon as https://github.com/shopware/shopware/pull/5925 is merged and deployed.